src/main/scala/translating/ILtoIL.scala

```diff
@@ -0,0 +1,279 @@
+package translating
+import ir._
+
+
+private class ILSerialiser extends Visitor {
```

Comment:
probably better to extend ReadOnlyVisitor since this isn't editing the IR?

Reply:
Thanks :facepalm: